src/PasswordManagerApp.java

```diff
@@ -0,0 +1,18 @@
+package src;
+
+//import src.usecases.ComputePasswordStrength;
+
+import src.controller.StrengthCheckerUIControl;
+import src.controller.UIController;
+
+public class PasswordManagerApp {
+    static void ComputePasswordStrength() {
```

Comment:
Method names should always start with a lowercase letter: computePasswordStrength.